build-system/amp.extern.js

```diff
@@ -35,7 +35,23 @@ window.AMP = {};
 
 // Externed explicitly because we do not export Class shaped names
 // by default.
-window.AMP.BaseElement;
+/**
+ * @constructor
+ */
+window.AMP.BaseElement = function(element) {};
+
+window.AMP.BaseElement.prototype.layout_;
```

Comment:
@dvoytenko This is special because the extensions subclass a class from a different compilation unit.